### PR TITLE
Add ECPoint class to devpack

### LINKED ISSUE #285: Add an `ECPoint` class
Add an `ECPoint` class to the devpack. 

See https://github.com/neo-project/neo-devpack-dotnet/pull/362/files#diff-d6c9a6e8f7b14263821a1de3af3a76a5d839e9874d5a2fdce8e307d67be2b6fc)

### PR DESCRIPTION
**Pull request by csmuller**
Closes #285.

### CODE REVIEW

**Comment by csmuller** [comment]
Moving to next milestone according to its corresponding milestone.

File: compiler/src/test-integration/java/io/neow3j/compiler/ECPointIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    public void createEcPointFromValidByteArray() throws IOException {
+        String publicKey = "03b4af8d061b6b320cce6c63bc4ec7894dce107bfc5f5ef5c68a93b4ad1e136816";
+        NeoInvokeFunction response = callInvokeFunction(byteArray(publicKey));
+        assertThat(response.getInvocationResult().getStack().get(0).asByteString().getAsHexString(),
+                is(publicKey));
+    }
+
+    @Test
+    public void createEcPointFromInvalidByteArray() throws IOException {
+        // Public key byte array is one byte to short
```
Comment (gsmachado) [inline compiler/src/test-integration/java/io/neow3j/compiler/ECPointIntegrationTest.java:35]:
```suggestion
        // Public key byte array is one byte too short
```

File: compiler/src/test-integration/java/io/neow3j/compiler/ECPointIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    public void createEcPointFromValidString() throws IOException {
+        String publicKey = "03b4af8d061b6b320cce6c63bc4ec7894dce107bfc5f5ef5c68a93b4ad1e136816";
+        NeoInvokeFunction response = callInvokeFunction(byteArray(publicKey));
+        assertThat(response.getInvocationResult().getStack().get(0).asByteString().getAsHexString(),
+                is(publicKey));
+    }
+
+    @Test
+    public void createEcPointFromInvalidString() throws IOException {
+        // Public key byte array is one byte to short
```
Comment (gsmachado) [inline compiler/src/test-integration/java/io/neow3j/compiler/ECPointIntegrationTest.java:51]:
```suggestion
        // Public key byte array is one byte too short
```

**Review by gsmachado — APPROVED** [review]
Just very minor comments. It's all good!

However, I could not get how the test coverage decreased -0.03% according to codecov. But, anyway.

**Pull request merged**